src/Controls/src/Core/Layout/FlexLayout.cs

```diff
@@ -0,0 +1,533 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using Microsoft.Maui.Controls.Internals;
+using Microsoft.Maui.Layouts;
+using Flex = Microsoft.Maui.Layouts.Flex;
+
+// This is a temporary namespace until we rename everything and move the legacy layouts
+namespace Microsoft.Maui.Controls.Layout2
```

Comment:
If we move it to Core, can we remove the Controls version and directly use Microsoft.Maui.Layouts.FlexLayout?

Reply:
If we think the new version is functional and stable enough, then we can move it into the Layout namespace and remove the old version. 
We still haven't fully decided whether the old layouts will be moving into a compatibility package or staying in Controls.